sql-db/hibernate-fulltext-search/src/test/java/io/quarkus/qe/hibernate/search/OpenShiftPostgresqlMultitenantNoPanacheIT.java

```diff
@@ -0,0 +1,37 @@
+package io.quarkus.qe.hibernate.search;
+
+import org.junit.jupiter.api.condition.EnabledIfSystemProperty;
+
+import io.quarkus.test.bootstrap.DefaultService;
+import io.quarkus.test.bootstrap.PostgresqlService;
+import io.quarkus.test.bootstrap.RestService;
+import io.quarkus.test.scenarios.OpenShiftScenario;
+import io.quarkus.test.services.Container;
+import io.quarkus.test.services.QuarkusApplication;
+
+@OpenShiftScenario
+@EnabledIfSystemProperty(named = "ts.redhat.registry.enabled", matches = "true")
+public class OpenShiftPostgresqlMultitenantNoPanacheIT extends AbstractMultitenantNoPanacheIT {
+    static final int ELASTIC_PORT = 9200;
+    static final int POSTGRESQL_PORT = 5432;
+
+    @Container(image = "docker.elastic.co/elasticsearch/elasticsearch-oss:7.10.0", port = ELASTIC_PORT, expectedLog = "started")
```

Comment:
This container is used in multiple places. Perhaps it should be parameterized by properties like the other commonly used container names?